programs/mango-v4/Cargo.toml

```diff
@@ -29,7 +29,7 @@ anchor-lang = { workspace = true }
 anchor-spl = { workspace = true }
 arrayref = "0.3.6"
 bincode = "1.3.3"
-borsh = { version = "0.9.3", features = ["const-generics"] }
+borsh = { version = "0.10.3", features = ["const-generics"] }
```

Comment:
fwiw, I think const-generics support is default in borsh 0.10, can drop the features specification